must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -49,13 +52,19 @@ commands_desc+=("logicalvolume")
 commands_desc+=("lvmvolumegroup")
 commands_desc+=("lvmvolumegroupsnodestatus")
 commands_desc+=("pods")
+commands_desc+=("volumesnapshot -A")
```

Comment:
Are the volumeSnapshot resources created in the same namespace as the PVC? If yes, this will not be the openshift-storage namespace.

Reply:
Fixed